README.rst

```diff
@@ -55,7 +55,14 @@ Run the command
 
   python two_implies.py
 
-The implies function in logic has the following truth table:
+Here is a definition of the implies function, from Wikipedia:
+
+The concept of logical implication is associated with an operation on two 
+logical values, typically the values of two propositions, that produces a 
+value of false just in case the first operand is true and the second operand
+is false.
```

Comment:
I don't think this is helpful to someone who is confused by this. Perhaps just link the `implies function` to the wikipedia link. I understood what was happening after reading up on material condition. 

I also think it would help if rather than `result`, you put `X -> Y` as that should be more familiar notation.